public/content/documentation/web/component/password-input.md

```diff
@@ -0,0 +1,218 @@
+## General Notes
+
+How to test a password input
+
+## Condensed
+
+### #a11y - Web Accessibility Acceptance Criteria
+
+How to test a header
+
+1. Test keyboard only, then screen reader + keyboard actions
+   - Tab: Focus moves visibly to the input or show password checkbox
+   - Spacebar: Toggles the show password checkbox
+
+2. Test mobile screenreader gestures
+   - Swipe: Focus moves to the input
+   - Keyboard: Keyboard appears
+
+3. Listen to screenreader output on all devices
+   - Name: Its purpose is clear
+   - Role: It identifies itself as a text input
+   - Group: Hints or errors are read after the label (Ex: Password formatting)
+   - State: It expresses if the password is being shown and if applicable: required, disabled / dimmed / unavailable
+
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/password-input](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/password-input)
+
+## Gherkin
+
+### #a11y - Web Accessibility Acceptance Criteria
+
+How to test a header
+
+GIVEN THAT I am on a page with a header landmark
+
+1. Keyboard for mobile & desktop
+
+   - WHEN I use the tab key to move focus to the password input I SEE focus is strongly visually indicated
+   - THEN when I use the tab key to move focus to the show/hide password feature I SEE its name, role and state
+   - THEN when I use the show/hide password feature I SEE the state of the password visibility (with or without characters entered)
+
+2. Desktop screenreader
+   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND I use the tab key to move focus to the password input
+     - I HEAR Its purpose is clear
```

Comment:
After "I HEAR" on this line and the three below, let's update the capital letters to lowercase ("I" in "Its", "I" in "It" twice, and "H" in "Hints").